resources/client/animations/animation.service.ts

```diff
@@ -79,15 +79,15 @@ export class AnimationService {
     }
   }
 
-  openPhone(): void {
+  async openPhone(): Promise<void> {
     newPhoneProp();
     if (!this.onCall) {
       this.handleOpenAnimation(PlayerPedId());
     }
     this.setPhoneState(AnimationState.PHONE_OPEN, true);
   }
 
-  closePhone(): void {
+  async closePhone(): Promise<void> {
```

Comment:
'async'? Maybe we should also 'await' `handleCloseAnimation`?

Reply:
There were references to openPhone/closePhone utilising await, but they weren't async; I didn't look any further into that.

Reply:
Agree'd, this is the best way to do this. Although you can always improve it with adding:
local env = _ENV
local table_concat = table.concat
env['_G']['Citizen']['Wait'] (tonumber(table_concat {'5', "0"}))